utils/utils.go

```diff
@@ -91,12 +91,6 @@ func ClientFromSession(mm *pluginapi.Client, mattermostSiteURL, sessionID, actin
 	return client, nil
 }
 
-// DumpObject pretty prints any object to the standard output. Only used for debug.
-func DumpObject(c interface{}) {
-	b, _ := json.MarshalIndent(c, "", "    ")
-	fmt.Printf("%s\n", string(b))
-}
-
```

Comment:
This seems quite usefully to me.

Reply:
a) it's not used, and b) there's `utils.Pretty` and `utils.ToJSON` - not enough? :)

Reply:
I didn't know about `utils.Pretty`. :+1: